pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -669,16 +662,12 @@ void Panels::MemoryCardListPanel_Simple::UiCreateNewCard(McdSlotItem& card)
 	if (result != wxID_CANCEL)
 	{
 		card.IsEnabled = true;
-		card.Filename = dialog.result_createdMcdFilename;
+		card.Filename  = Path::FromWxString(dialog.result_createdMcdFilename);
```

Comment:
Unnecessary extra space. You don't need to align the `=` with the lines above and below.